ucsmsdk/utils/ucsguilaunch.py

```diff
@@ -56,7 +56,7 @@ def ucs_gui_launch(handle, need_url=False):
 
     jnlp_file = None
     try:
-        ucsm_gui_url = "%s/ucsm/ucsm.jnlp" % handle.uri
```

Comment:
How is this change related to this PR or firmware upload in general?

Reply:
This is an another issue -
https://github.com/CiscoUcs/ucsmsdk/issues/241
https://jira-eng-sjc11.cisco.com/jira/browse/UCSES-27 (Please refer attachment)

Does it required to create a new Fork and raise PR ?

Reply:
You can create new branches in your fork for each bug fix and submit individual PRs from each of your bug fix branches. Do not mix multiple unrelated changes in a single PR.

Reply:
Sure,